### PR TITLE
more deletions

### PR DESCRIPTION
**Pull request by nrosed**
additional deletions focused on my sections

### CODE REVIEW

File: content/01.main_content.md [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
+  While a secondary audit performed by LaFrance showed no improvement, Yong, through continuous self-auditing, was able to achieve gender parity in his reporting.
 
-In award recognition, an audit of the International Society for Computational Biology (ISCB) honorees revealed a significant disparity against people with East Asian name origins and towards US-affiliated scientists.
-  After this study was made available to the public, the following set of honorees had the highest mean predicted probability of an honoree having an East Asian name of any previous year.
-  Additionally, the nominating committee inducted the first China-based fellow into ISCB [@doi:10.1016/j.cels.2021.07.007].
+In award recognition, an audit of the International Society for Computational Biology (ISCB) honorees revealed a significant under-representation of people with East Asian name origins and under-representation of US-affiliated scientists.
```
Comment (arielah) [inline content/01.main_content.md]: over-representation you mean?

File: content/01.main_content.md [hunk 2]
```diff
@@ -34,15 +33,14 @@ Proxy predictions of ethnicity via name origin are more difficult still; choosin
...
+  2. *Individuals know best.* Self-identified demographic information should be used in preference to algorithmic predictions.
 
   3. *Aggregates only.* Audit results should not affect individuals.
-  Gender expression and ethnicity of an individual are not for an algorithm to decide nor are they the focus of an audit. 
-  The analyses must focus only on aggregate estimations and any intermediary predictions on an individual should not be used independently.
-  In addition, analysts must be mindful of hidden subpopulations that may be obscured when calculating aggregate statistics.
+  Analyses must focus on aggregate estimations and any intermediary individual predictions should not be used externally.
+  In addition, analysts must be mindful of hidden subpopulations that may be obscured during aggregation.
+
+  4. *Inform the public.* Internal audits can help institutions reflect, however it does not give the public an opportunity to hold them accountable.
```
Comment (arielah) [inline content/01.main_content.md]: This sentence seems a little strange at the moment. What do you think of "While internal audits can help institutions reflect, the public must be given an opportunity to hold them accountable." ? It's one fewer word

**Review by arielah — APPROVED** [review]
Looks good, just one typo and one suggestion

File: content/01.main_content.md (hunk 1 above)
Comment (nrosed) [reply content/01.main_content.md]: oops, fixed: https://github.com/greenelab/disparities_opinion/pull/8/commits/4844c380c9cf2566cdf98c5c6c31833a8492cf7f

File: content/01.main_content.md (hunk 2 above)
Comment (nrosed) [reply content/01.main_content.md]: yes sounds better: https://github.com/greenelab/disparities_opinion/pull/8/commits/30b8a87587fddf7e522503ce07829c02240e7602

**Pull request merged**